src/app/lib/config/services/ws.ts

```diff
@@ -509,46 +563,55 @@ export const service: DefaultServiceConfig = {
             id: 'azeri',
             href: 'https://www.bbc.com/azeri',
             label: 'Azərbaycanca xəbərlər',
+            lang: 'az',
           },
           {
             id: 'news',
             href: 'https://www.bbc.co.uk/news',
             label: 'News in English',
+            lang: 'en',
           },
           {
             id: 'naidheachdan',
             href: 'https://www.bbc.co.uk/naidheachdan',
             label: 'Naidheachdan',
+            lang: 'gd',
           },
           {
             id: 'polska',
             href: 'https://www.bbc.com/polska',
             label: 'BBC News po polsku',
+            lang: 'pl',
           },
           {
             id: 'russian',
             href: 'https://www.bbc.com/russian',
             label: 'Новости на русском языке',
+            lang: 'ru',
           },
           {
             id: 'serbian',
             href: 'https://www.bbc.com/serbian',
             label: 'BBC News na srpskom',
+            lang: 'sr',
```

Comment:
Does this need to match the lang of the default variant?
```suggestion
            lang: 'sr-latn',
```